_docker/drupal/drupal-filesystem/web/themes/custom/rhd/templates/node--cheat-sheet.html.twig

```diff
@@ -0,0 +1,111 @@
+*   - node--promoted: Appears on nodes promoted to the front page.
+*   - node--sticky: Appears on nodes ordered above other non-sticky nodes in
+*     teaser listings.
+*   - node--unpublished: Appears on unpublished nodes visible only to site
+*     admins.
+* - title_attributes: Same as attributes, except applied to the main title
+*   tag that appears in the template.
+* - content_attributes: Same as attributes, except applied to the main
+*   content tag that appears in the template.
+* - author_attributes: Same as attributes, except applied to the author of
+*   the node tag that appears in the template.
+* - title_prefix: Additional output populated by modules, intended to be
+*   displayed in front of the main title tag that appears in the template.
+* - title_suffix: Additional output populated by modules, intended to be
+*   displayed after the main title tag that appears in the template.
+* - view_mode: View mode; for example, "teaser" or "full".
+* - teaser: Flag for the teaser state. Will be true if view_mode is 'teaser'.
+* - page: Flag for the full page state. Will be true if view_mode is 'full'.
+* - readmore: Flag for more state. Will be true if the teaser content of the
+*   node cannot hold the main body content.
+* - logged_in: Flag for authenticated user status. Will be true when the
+*   current user is a logged-in member.
+* - is_admin: Flag for admin user status. Will be true when the current user
+*   is an administrator.
+*
+* @see template_preprocess_node()
+*
+* @todo Remove the id attribute (or make it a class), because if that gets
+*   rendered twice on a page this is invalid CSS for example: two lists
+*   in different view modes.
+*
+* @ingroup themeable
+*/
+#}
+<div{{ attributes.addClass(["row", "content"]).setAttribute("id", "rhd-cheat-sheet") }}>
+  <div class="row">
+    <div class="large-24 medium-19 columns">
+        <h4 class="content-type">Cheat Sheet</h4>
+        {{ title_prefix }}
+        <h2{{ title_attributes.addClass(["cs-title"]) }}>{{ label }}</h2>
+        {{ title_suffix }}
+        <div class="cs-info">
+            <ul>
+                <li>
+                    <strong{{ author_attributes }}>By {{content.field_cheat_sheet_author}} </strong>
+                </li>
+                <li>
+                    <small>Published: {{ date }}</small>
+                </li>
+                <li class="right cs-social">
+                    {% include '@rhd/share.html.twig' %}
+                </li>
+            </ul>
+        </div>
+    </div>
+  </div>
+  <div class="row">
+    <div{{ content_attributes.addClass(["large-18", "medium-15", "columns"]) }}>
+        {{ content.body }}
+    </div>
+    <div class="large-6 medium-5 columns">
+        {% if content.field_cheat_sheet_tags|render is not empty %}
+            <strong>Tags:</strong> {{ content.field_cheat_sheet_tags }}
+        {% endif %}
+    </div>
+    <div {{ content_attributes.addClass(["large-20", "medium-15", "columns"]) }}>
```

Comment:
Is this intended to be a new row? If so, we should setup the div to be within a new row div.